frontend/tests/_bootstrap.php

```diff
@@ -7,3 +7,7 @@
 require_once YII_APP_BASE_PATH . '/vendor/yiisoft/yii2/Yii.php';
 require_once YII_APP_BASE_PATH . '/common/config/bootstrap.php';
 require_once __DIR__ . '/../config/bootstrap.php';
+
+//codeception namespace
+\Yii::setAlias('@frontend/tests/Helper', '@frontend/tests/_support/Helper');
+\Yii::setAlias('@frontend/stub', '@frontend/tests/_stub');
```

Comment:
Shouldn't namespace be `@frontend/tests/stub`?

Reply:
Agree.